lib/oktakit/client.rb

```diff
@@ -31,12 +31,17 @@ class Client
       builder.adapter Faraday.default_adapter
     end
 
-    def initialize(token:, organization: nil, api_endpoint: nil)
+    def initialize(token: nil, access_token: nil, organization: nil, api_endpoint: nil)
       if organization.nil? && api_endpoint.nil?
         raise ArgumentError, "Please provide either the organization or the api_endpoint argument"
       end
 
+      if token.blank? && access_token.blank?
```

Comment:
It should be `nil?` over `blank?` as `blank?` is monkey patched in Rails ActiveSupport. We'd likely get an error if we won't pass these optional arguments.

Reply:
Good catch. I didn't know that. Thanks. But we won't get an error because we're using it inside a Rails application. But I'll go ahead and fix it as this gem is not meanly for only Rails. Thanks once again.